src/test/java/org/broadinstitute/hellbender/tools/spark/sv/discovery/DiscoverVariantsFromContigAlignmentsSAMSparkUnitTest.java

```diff
@@ -136,9 +136,7 @@ public void testProcessEvidenceTargetLinks(final List<EvidenceTargetLink> etls,
                         metadata, inputVariants, referenceMultiSource);
 
         Assert.assertEquals(processedVariantContexts.size(), expectedVariants.size());
-        for (int i = 0; i < processedVariantContexts.size(); i++) {
-            VariantContextTestUtils.assertEqualVariants(processedVariantContexts, expectedVariants);
-        }
+        VariantContextTestUtils.assertEqualVariants(processedVariantContexts, expectedVariants);
```

Comment:
extremely trivial: I think `VariantContextTestUtils.assertEqualVariants` tests for size equality first, so the line above is not necessary any more.